odoo/cli/populate.py

```diff
@@ -10,19 +10,38 @@
 from . import Command
 _logger = logging.getLogger(__name__)
 
+# NB: sibling options (same group) are still valid but not shown by --help,
+#     other groups are completely removed
+STANDARD_OPTIONS_TO_KEEP = {
+    '--config', '--database', '--addons-path',
+    '--log-level', '--log-handler', '--logfile',
```

Comment:
+ --log-sql ?
Is there a way to keep the whole option group of logging config ?

Reply:
>  Is there a way to keep the whole option group of logging config ?

The entire group is always kept as long as there are visible options in it, the unselected options are just hidden.